app/AppLoader.scala

```diff
@@ -16,12 +21,25 @@ class AppLoader extends ApplicationLoader {
      * The JDBC driver interprets these using the JVM's default timezone which is
      * almost certainly what no sane person ever wants to do.
      */
-
     System.setProperty("user.timezone", "UTC")
     TimeZone.setDefault(TimeZone.getTimeZone("UTC"))
 
-    val contextWithConfiguration = ConfigurationLoader.playContext("editorial-production-metrics", context)
-    new AppComponents(contextWithConfiguration).application
+    val defaultAppName = "editorial-production-metrics"
+    val identity = AppIdentity.whoAmI(defaultAppName)
+
+    val awsCredentials: AwsCredentialsProvider = identity match {
+      case _: DevIdentity => ProfileCredentialsProvider.create("composer")
+      case _ => DefaultCredentialsProvider.create()
+    }
+
+    val loadedConfig = ConfigurationLoader.load(identity, awsCredentials) {
```

Comment:
Example of this being used in another of our projects can be found here: https://github.com/guardian/typerighter/blob/1cbfe7f5a8b33ef6cf7c69412a330c6d174a84e4/apps/rule-manager/app/AppLoader.scala

Unfortunately, it is necessary to load additional config here rather than in `AppConfig.scala` as the config is required by several of the traits extended by `AppComponents.scala` instantiated below